src/renderer/viz/expressions.js

```diff
@@ -120,9 +137,291 @@
  * More information in {@link carto.expressions.palettes|carto.expressions.palettes}.
  *
  * @typedef {} Palette
+ * @name Palette
+ * @memberof Types
+ * @api
+ */
+
+/**
+ * Type of Image Expressions.
+ *
+ * More information in {@link carto.expressions.image|carto.expressions.image}.
+ *
+ * @typedef {} Image
+ * @name Image
+ * @memberof Types
  * @api
  */
 
+/**
+ * Type of Translate Expressions.
+ *
+ * More information in {@link carto.expressions.translate|carto.expressions.translate}.
+ *
+ * @typedef {} Transformation
+ * @name Transformation
+ * @memberof Types
+ * @api
+ */
+
+/**
+ * @namespace Constants
```

Comment:
Why do we have another namespace with the expressions duplicated?

Reply:
They are not duplicated, these are not expressions but types. This is meant to list the available types.